### PR TITLE
Build updates

### PR DESCRIPTION
**Pull request by rolyatmax**
Splitting out from #80

### CODE REVIEW

**Comment by rolyatmax** [comment]
@AlanFoster Take a look and let me know what you think!

File: package.json [hunk 1]
```diff
@@ -45,7 +49,7 @@
     "gulp-connect": "^2.2.0",
     "gulp-ext-replace": "^0.1.0",
     "gulp-gh-pages": "^0.4.0",
-    "gulp-sass": "^1.2.0",
+    "gulp-sass": "^2.0.4",
```
Comment (AlanFoster) [inline package.json]: Should this be left as part of #79 ?

Comment (rolyatmax) [inline package.json]: I think this should be part of this PR as it relates to fixing a problem with the build process and not really to updating to React 0.14

**Comment by AlanFoster** [comment]
LGTM; Thanks for the PR! :)

**Pull request merged**